ceph_cfg/mdl_updater.py

```diff
@@ -227,31 +223,77 @@ def partitions_all_refresh(self):
         self.model.part_pairent = part_map
 
 
-    def partition_table_refresh(self):
-        for disk_name in self.model.lsblk.keys():
-            arguments = [
-                'parted',
-                disk_name,
-                'print'
-                ]
-            log.debug("Running:%s" % (" ".join(arguments)))
-            output = utils.execute_local_command(arguments)
-            if output["retcode"] != 0:
-                raise Error("Failed executing '%s' Error rc=%s, stdout=%s stderr=%s" % (
-                    " ".join(arguments),
-                    output["retcode"],
-                    output["stdout"],
-                    output["stderr"]
-                    ))
-            part_type = None
-            for line in output["stdout"].split('\n'):
-                split_line = line.split(':')
-                if split_line[0] != 'Partition Table':
+    def partitions_all_refresh_parted(self):
```

Comment:
I'm making my way through the commits, but partitions_all_refresh_parted does not exactly roll off the tongue.  Maybe partitions_refresh_all unless you have another somewhere.